deployment/ansible/roles/cac-tripplanner.app/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Install packages
+  apt: name={{ item }} state=present
+  with_items:
+    - postgresql-9.3
+    - postgresql-9.3-postgis-2.1
+    - libpq-dev
+    - python-dev
+    - python-psycopg2
+
+- name: Install pip packages
+  pip: name={{ item }}
+       virtualenv=/app_env
+  with_items:
```

Comment:
It's been working on HunchLab for a week or so. We've been using the same approach for ruby gems for a while now and that has been good.

It definitely makes packer integration easier since it doesn't require pushing up the requirements.txt file to the instance.